apps/src/netsim/DashboardUser.js

```diff
@@ -48,6 +48,21 @@ var DashboardUser = module.exports = function () {
    * @type {string}
    */
   this.name = "";
+
+  /**
+   * Whether the user is a site admin.
+   * NOTE: This should only be used in a convenience/display sense; anything
+   *       requiring actual security should be authenticated through the server,
+   *       and not depend on client code.
+   * @type {boolean}
+   */
+  this.isAdmin = false;
+
+  /**
+   * List of sections owned by this user.
+   * @type {Array}
```

Comment:
either `Array.<Section>` or `Section[]` plus a `@typedef Section` would be nice here.